src/celluloid-application.c

```diff
@@ -307,6 +308,49 @@ options_handler(GApplication *gapp, GVariantDict *options, gpointer data)
 	return version?0:-1;
 }
 
+static gboolean
+local_cmdline_handler (	GApplication *gapp,
+			gchar ***arguments,
+			gint *exit_status)
+{
+	CelluloidApplication *app = CELLULOID_APPLICATION(gapp);
+	gchar **argv = *arguments;
+	g_clear_pointer(&app->mpv_options, g_free);
+
+	GPtrArray *options = g_ptr_array_new();
```

Comment:
We need to set `free_func` or `g_ptr_array_free()` won't free the contents of the array.

```suggestion
	GPtrArray *options = g_ptr_array_new_with_free_func(g_free);
```